parts/k8s/cloud-init/artifacts/sshd_config

```diff
@@ -48,15 +61,33 @@ PermitEmptyPasswords no
 ChallengeResponseAuthentication no
 
 # Change to no to disable tunnelled clear text passwords
-PasswordAuthentication no
+#PasswordAuthentication yes
```

Comment:
This implies we're using the default of `yes` now. Is that what CIS wants? Seems less safe.

Reply:
@jackfrancis any thoughts? I took the file as-is but it's true that this particular instance seems to be less safe

Reply:
Agree with Matt, let's not include this change.